modules/imgcodecs/src/grfmt_exr.cpp

```diff
@@ -691,6 +691,10 @@ bool  ExrEncoder::write( const Mat& img, const std::vector<int>& params )
                 CV_Error(Error::StsBadArg, "IMWRITE_EXR_COMPRESSION is invalid or not supported");
             }
         }
+        if (params[i] == IMWRITE_EXR_DWA_COMPRESSION_LEVEL)
+        {
+            header.dwaCompressionLevel() = params[i + 1];
```

Comment:
Need to add compile-time version check here (through `#if`)

Reply:
Oops, looks like the setting itself is available only from version 3 on.

Reply:
Should be ok now.